deploy/charts/venafi-kubernetes-agent/README.md

```diff
@@ -0,0 +1,162 @@
+  - Under "Service Account" click that drop down and select the previously created service account.
+  - Then check the "The connection command has completed." box and select "continue".
+- On step 5, either wait for validation or select "Finish" to go back to the cluster list.
+
+### 5) Deployment Verification
+
+Check the agent logs to ensure you see a similar entry to the following:
+
+```console
+2023/10/24 12:10:03 Running Agent...
+2023/10/24 12:10:03 Posting data to: https://api.venafi.cloud/
+2023/10/24 12:10:04 Data sent successfully.
+```
+
+You can do this with the following command:
+
+```shell
+kubectl logs -n ${VENAFI_NAMESPACE} $(kubectl get pod -n ${VENAFI_NAMESPACE} -l app.kubernetes.io/instance=agent -o jsonpath='{.items[0].metadata.name}')
+```
+
+You can also check inb the Venafi Control Plane to see when the "Last Check In" was for your cluster.
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| affinity | object | `{}` |  |
+| authentication | object | `{"secretKey":"privatekey.pem","secretName":"agent-credentials"}` | Authentication details for the Venafi Kuberente Agent |
+| authentication.secretKey | string | `"privatekey.pem"` | Key name in the references secret |
+| authentication.secretName | string | `"agent-credentials"` | Name of the secret containing the privatekey |
+| command | list | `[]` | Specify the command to run overriding default |
+| config | object | `{"clientId":"","configmap":{"key":null,"name":null},"period":"0h1m0s","server":"https://api.venafi.cloud/"}` | Configuration section for the Venafi Kubernetes Agent itself |
+| config.configmap | object | `{"key":null,"name":null}` | Sepcify ConfigMap details to load config from an existing resource This should be blankby default unless you have you own config |
+| config.period | string | `"0h1m0s"` | Send data back to the platform every minute unless changed |
+| config.server | string | `"https://api.venafi.cloud/"` | Overrides the server if using a proxy in your environment |
+| extraArgs | list | `[]` | Specify additional argument to pass to the agent |
+| fullnameOverride | string | `""` | Helm default setting, use this to shorten install name |
+| image.pullPolicy | string | `"IfNotPresent"` | Defaults to only pull if not already present |
+| image.repository | string | `"quay.io/jetstack/preflight"` | Default to Open Source image repository |
+| image.tag | string | `"v0.1.43"` | Overrides the image tag whose default is the chart appVersion |
+| imagePullSecrets | list | `[]` | Specify image pull credentials if using a prviate registry |
+| nameOverride | string | `""` | Helm default setting to override release name, leave blank |
+| nodeSelector | object | `{}` |  |
+| podAnnotations | object | `{}` |  |
+| podSecurityContext | object | `{}` |  |
+| replicaCount | int | `1` | default replicas, do not scale up |
+| resources.limits.cpu | string | `"500m"` |  |
```

Comment:
I have had many discussions with Raf and if you cared about maximum performance vs predictability you wouldn't have the CPU limit, everything else looks fine.. i would be curious on an average openshift cluster if our memory requests/limits are matching.

Reply:
Perhaps I can add this to my next OpenShift test cluster and see what it does by default.
I think it's fine for resourcing to expect people to set their own requirements.
Also I took the setting from the install YAML passed to me, so assume this is what `venctl` would generate as well.

Reply:
I re-read the previous lines and then disagreed. I've added defaults but people are free to override at install time to desired values. Or `null` values for that matter as you say in limits.